src/components/markdown.component.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { WebView, Platform } from 'react-native';
+
+export class MarkDown extends Component {
```

Comment:
Should probably be named `Markdown`.

Reply:
Renamed to MarkdownWebview in last commit